SPEC.md

```diff
@@ -539,6 +542,35 @@ The frame bookend is a simple event that can be used to determine that the entir
 | 0x1 | Frame Number | int32 | The number of the frame. Starts at -123. Frame 0 is when the timer starts counting down | 3.0.0
 | 0x5 | Latest Finalized Frame | int32 | For non-rollback, this should always equal the frame number. For rollback, this indicates the index of a frame which is guaranteed not to happen again (finalized). This is very useful when reading a file in real-time to make sure you are processing "finalized" information | 3.7.0
 
+### FOD Platforms
+This event only occurs on Fountain of Dreams, and is sent every frame the height of the platform is changing.
+
+| Offset | Name | Type | Description | Added |
+| --- | --- | --- | --- | --- |
+| 0x0 | Command Byte | uint8 | (0x3F) The command byte for the FOD platform event | 3.17.0
+| 0x1 | Frame Number | int32 | The number of the frame. Starts at -123. Frame 0 is when the timer starts counting down | 3.17.0
+| 0x5 | Platform | uint8 | Which platform has moved. (0 = Right, 1 = Left) | 3.17.0
+| 0x6 | Height | float | The platform's new height | 3.17.0
+
+### Whispy Blow Direction
+This event only occurs on Dreamland 64, and is sent whenever Whispy changes blow directions.
+
+| Offset | Name | Type | Description | Added |
+| --- | --- | --- | --- | --- |
+| 0x0 | Command Byte | uint8 | (0x40) The command byte for the blow direction event | 3.17.0
+| 0x1 | Frame Number | int32 | The number of the frame. Starts at -123. Frame 0 is when the timer starts counting down | 3.17.0
+| 0x5 | Direction | uint8 | Which direction Whispy is blowing (0 = None, 1 = Left, 2 = Right) | 3.17.0
+
+### Stadium Transformations
+This event only occurs on Pokemon Stadium, and is sent whenever the transformation event changes.
```

Comment:
I think technically the event is sent whenever the transformation event changes _or_ the transformation type changes